src/components/Audio.js

```diff
@@ -0,0 +1,20 @@
+import React, {useState} from 'react';
+import mute from '../icons/mute.svg';
+import unmute from '../icons/unmute.svg';
+
+function Audio() {
```

Comment:
Referring to comment below--you can pass down the content from this line in App.js (`<Audio onClick={toggler} toggle={toggle} />`) by changing this to `function Audio( {onClick, toggle} )`